travis-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+dotnet test -c Debug -f netcoreapp1.0 MoreLinq.Test
+RID=`dotnet --info | grep RID | sed -E 's/ *RID: *(.+)/\1/'`
```

Comment:
[I used the same technique](https://github.com/morelinq/MoreLINQ/blob/c9e3c09b738ce162d4a90e8d16edcba4732ea46d/test.sh#L6) at first but it didn't work out well on Windows. While `dotnet --info` reported an RID of `win10-x64` or `win10-x86` on Windows 10, the test project was still built to a directory based on the RID of Windows 7. I ended up hard-coding it first in 7c0a0f9f03de696c9feb010adbc296c45ad586e8 (:speak_no_evil:) before picking it up dynamically in 7b8551c5b4bd30d6e3f67fd5d456b8ee7f4f1ec6 so it works for `win7-x64` or `win7-x86` depending on the bit-ness of the OS.